lib/src/setupTests.ts

```diff
@@ -6,9 +6,26 @@
  */
 /// <reference types="./__testutils__/jest-enzyme-overrides" />
 
-import 'jest-enzyme';
-
 import Enzyme from 'enzyme';
 import Adapter from 'enzyme-adapter-react-16';
+import { runTimers } from './__testutils__/rtlUtils';
 
 Enzyme.configure({ adapter: new Adapter() });
+
+beforeEach(function() {
+  // because so many of our components rely on NxTooltip which initializes asynchronously. If some tests
+  // use fake timers and some don't it can cause weird inter-test timing bugs
+  jest.useFakeTimers();
+
+  // JSDOM is missing this function. https://github.com/jsdom/jsdom/issues/3002
+  Range.prototype.getBoundingClientRect = jest.fn().mockReturnValue({
```

Comment:
With this now in `setupTests.ts`, we should be able to remove it from `NxBreadcrumb` and `NxStatefulBreadcrumb`

Reply:
> With this now in setupTests.ts, we should be able to remove it from NxBreadcrumb and NxStatefulBreadcrumb

[b2ed92d](https://github.com/sonatype/sonatype-react-shared-components/pull/893/commits/b2ed92dd6102df2564dc69cb103d08fd89ec40c9)